2023-06-23 15:41:13,823 - [INFO] - orders_service.py[line:65]: 元数据表orders_processed_files不存在，现已创建
2023-06-23 15:41:13,899 - [INFO] - orders_service.py[line:42]: 本次需要采集的文件为['D:/资料/etl项目/数据资料/JSON数据（订单业务）/x00', 'D:/资料/etl项目/数据资料/JSON数据（订单业务）/x01', 'D:/资料/etl项目/数据资料/JSON数据（订单业务）/x06', 'D:/资料/etl项目/数据资料/JSON数据（订单业务）/x07', 'D:/资料/etl项目/数据资料/JSON数据（订单业务）/x08', 'D:/资料/etl项目/数据资料/JSON数据（订单业务）/x09', 'D:/资料/etl项目/数据资料/JSON数据（订单业务）/x11', 'D:/资料/etl项目/数据资料/JSON数据（订单业务）/x20', 'D:/资料/etl项目/数据资料/JSON数据（订单业务）/x27', 'D:/资料/etl项目/数据资料/JSON数据（订单业务）/x30', 'D:/资料/etl项目/数据资料/JSON数据（订单业务）/x31', 'D:/资料/etl项目/数据资料/JSON数据（订单业务）/x36', 'D:/资料/etl项目/数据资料/JSON数据（订单业务）/x37', 'D:/资料/etl项目/数据资料/JSON数据（订单业务）/x38', 'D:/资料/etl项目/数据资料/JSON数据（订单业务）/x39', 'D:/资料/etl项目/数据资料/JSON数据（订单业务）/x43', 'D:/资料/etl项目/数据资料/JSON数据（订单业务）/x44', 'D:/资料/etl项目/数据资料/JSON数据（订单业务）/x52', 'D:/资料/etl项目/数据资料/JSON数据（订单业务）/x53', 'D:/资料/etl项目/数据资料/JSON数据（订单业务）/x54', 'D:/资料/etl项目/数据资料/JSON数据（订单业务）/x55', 'D:/资料/etl项目/数据资料/JSON数据（订单业务）/x62', 'D:/资料/etl项目/数据资料/JSON数据（订单业务）/x63', 'D:/资料/etl项目/数据资料/JSON数据（订单业务）/x65', 'D:/资料/etl项目/数据资料/JSON数据（订单业务）/x74', 'D:/资料/etl项目/数据资料/JSON数据（订单业务）/x80', 'D:/资料/etl项目/数据资料/JSON数据（订单业务）/x87', 'D:/资料/etl项目/数据资料/JSON数据（订单业务）/x88', 'D:/资料/etl项目/数据资料/JSON数据（订单业务）/x89']，共29个文件
2023-06-23 15:42:14,600 - [INFO] - orders_service.py[line:114]: 共向D:\PycharmProjects\standardisation\out_data\orders_20230623_154113.csv.tmp表插入29696条数据。
共向D:\PycharmProjects\standardisation\out_data\orders_detail_20230623_154113.csv.tmp表插入49440条数据
2023-06-23 15:42:14,931 - [INFO] - orders_service.py[line:144]: 共向D:\PycharmProjects\standardisation\out_data文件插入29696条数据。
共向orders_detail文件插入49440条数据
2023-06-23 15:42:15,713 - [WARNING] - mysql_util.py[line:88]: 表orders_processed_files已存在，创建失败！
2023-06-23 15:42:15,944 - [INFO] - orders_service.py[line:193]: 共向元数据插入29条数据
